code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -685,3 +762,148 @@
+///Updates or cancels stealth
+/datum/action/ability/xeno_action/stealth/phaseout/handle_stealth()
+	var/mob/living/carbon/xenomorph/xenoowner = owner
+	xenoowner.alpha = HUNTER_STEALTH_STILL_ALPHA * stealth_alpha_multiplier // instant full stealth regardless of movement.
+
+// ***************************************
+// *********** Death Mark
+// ***************************************
+/datum/action/ability/activable/xeno/hunter_mark/assassin
+	name = "Death Mark"
+	action_icon_state = "death_mark"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Psionically disturb a creature for 15 seconds, allowing you to deal double sneak attack damage and stun from it. They will know you are coming for them."
+	ability_cost = 50
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_HUNTER_MARK,
+	)
+	cooldown_duration = 30 SECONDS
+	require_los = FALSE
+	timeout = 15 SECONDS
+	warntarget = TRUE
+	chargeup = 2 SECONDS
+
+// ***************************************
+// *********** Displacement
+// ***************************************
+
+/datum/action/ability/xeno_action/displacement
+	name = "Displacement"
+	action_icon_state = "hunter_invisibility"
+	action_icon = 'icons/Xeno/actions/hunter.dmi'
+	desc = "Physically disappear, become incorporeal until you decide to reappear somewhere else, reappearing on lighted areas will disorient you and flicker the lights."
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOMORPH_HUNTER_DISPLACEMENT,
+	)
+	use_state_flags = ABILITY_USE_SOLIDOBJECT
+
+/datum/action/ability/xeno_action/displacement/action_activate()
+	var/mob/living/carbon/xenomorph/xenomorph_owner = owner
+	xenomorph_owner.change_form()
+
+/mob/living/carbon/xenomorph/hunter/assassin/change_form()
+	if(!loc_weeds_type)
+		balloon_alert(src, "We need to be on weeds.")
+		return
+	wound_overlay.icon_state = "none"
+	if(status_flags & INCORPOREAL) //will alert if the xeno will get disoriented due lit turf, if phasing in.
+		var/turf/whereweat = get_turf(src)
+		if(whereweat.get_lumcount() > 0.2) //is it a lit turf
+			balloon_alert(src, "We will be disoriented and sensed in this light.") //so its more visible to xeno.
+			//Marines can sense the manifestation if it's in lit-enough turf nearby.
+			visible_message(span_highdanger("Something begins to manifest nearby!"), span_xenohighdanger("We begin to manifest in the light... talls sense us!"))
+	if(do_after(src, 3 SECONDS, IGNORE_HELD_ITEM, src, BUSY_ICON_BAR, NONE, PROGRESS_GENERIC)) //dont move
+		do_change_form()
+
+///Finish the form changing of the hunter and give the needed stats
+/mob/living/carbon/xenomorph/hunter/assassin/proc/do_change_form()
+	playsound(get_turf(src), 'sound/effects/alien/new_larva.ogg', 25, 0, 1)
+	if(status_flags & INCORPOREAL)
+		var/turf/whereweat = get_turf(src)
+		if(whereweat.get_lumcount() > 0.2) //is it a lit turf
+			balloon_alert(src, "Light disorients us!")
+			adjust_stagger(2 SECONDS)
+			adjust_slowdown(2)
```

Comment:
add_slowdown() instead of adjust_slowdown, otherwise hunters are getting perma slowed down for emerging in light

Reply:
ah, odd.